recipes/sphinx-jinja/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "sphinx-jinja" %}
+{% set version = "1.1.0" %}
```

Comment:
use 1.1.1

Reply:
@tardyp Will do! Thanks for the new release!